docs/IR.md

```diff
@@ -351,7 +351,7 @@ Dimension variables are scoped to the declaration that they appear in. For graph
 
```

Comment:
Even the preceding paragraph seems inaccurate. This could be a subject for design discussion, but I believe currently the dimension variables are globally scoped.

Reply:
how should we follow up?

Reply:
Let's merge in what you have. I can take a shot at clarifying this.